docs/model_management.md

```diff
@@ -129,3 +135,34 @@ changes:
   *label_filename* property of the output it corresponds to in the
   [model configuration](model_configuration.md) must be performed at
   the same time.
+
+## Modifying the Model Repository
+
+Each model in a model repository [resides in its own
+sub-directory](model_repository.md#repository-layout). The activity
+allowed on the contents of a model's sub-directory varies depending on
+how Triton is using that model. The loaded or unloaded state of a
```

Comment:
How about just "The state of a model ... " since loaded/unloaded are not the only two states, Additionally loaded is represented by READY and unloaded by UNAVAILABLE. 
Additionally I do not think model metadata returns model state anymore. There is only model ready right?

Reply:
Model metadata will return error if the model isn't loaded.

Reply:
Yes you're correct.